app/src/main/java/io/pumpkinz/pumpkinreader/NewsListFragment.java

```diff
@@ -69,13 +74,12 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         layoutManager.setOrientation(LinearLayoutManager.VERTICAL);
         newsList.setLayoutManager(layoutManager);
 
-        dataset = new ArrayList<>();
-        newsAdapter = new NewsAdapter(dataset);
-        newsList.setAdapter(newsAdapter);
+        List<News> dataset = getMockData();
```

Comment:
Yes, the purpose of this is just to lay out the foundation of message passing between activities.
If you agree the general idea that we should use News and Comment model class, then the rest is just how do we wrap it.

So, it serves the purpose for this pull request.